.github/workflows/pythonpackage.yml

```diff
@@ -88,6 +85,10 @@ jobs:
     #     # flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
     #     # exit-zero treats all errors as warnings. The GitHub editor is 127 chars wide
     #     # flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
+    - name: Install libpcre3-dev
+      if: matrix.os == 'ubuntu-latest'
+      run: |
+        sudo apt update & sudo apt install -y libpcre3-dev
```

Comment:
we have `bin/install_pcre.sh` already just above: https://github.com/vaexio/vaex/blob/4c710c1fb117f5e63180404b9837c7deb803b832/.github/workflows/pythonpackage.yml#L52-L54

is it broken?

Reply:
see https://github.com/vaexio/vaex/actions/runs/12819691425/job/35747856845

`E   ImportError: libpcrecpp.so.0: cannot open shared object file: No such file or directory`

Reply:
I see. The wheels depend on that script: https://github.com/vaexio/vaex/blob/4c710c1fb117f5e63180404b9837c7deb803b832/.github/workflows/wheel.yml#L101

So maybe the wheels are broken and this smoke test doesn't catch it? https://github.com/vaexio/vaex/blob/4c710c1fb117f5e63180404b9837c7deb803b832/.github/workflows/wheel.yml#L107

What do you think about adding the apt-get to the install script? Maybe similar to this one, preferring the precompiled one over the source install? https://github.com/ahupp/python-magic/blob/65fb61c9c9aa6348bb95d1dd71b685720c2a8a23/add_libmagic.sh#L69

Reply:
I added it to the script. 
The source-built libraries are in /usr/local/lib. So LD_LIBRARY_PATH or runtime search path for the extension could be set otherwise.